functionalDB/chapter.md

```diff
@@ -1,103 +1,104 @@
-# Designing a database like an archaeologist
+<!-- American spelling --> 
+# CircleDB: An Archaeology-Inspired Database

```

Comment:
I suspect you wanted to save your excellent name as a surprise at the end, but we typically use the app name in the chapter title. I think the end still has impact because you explain why the name is so perfect.